docs/wiki/01_STUDY_PARAMETERS_SETUP/02_Clinical_Instruments/01_instrument_intro_prerequisites.md

```diff
@@ -0,0 +1,129 @@
+tested on a certain set of instruments if they are in cohort A and from site X, but 
+would be tested on different instruments if they are registered in cohort B at site Y. 
+For certain studies, the battery may also depend on the age of the participant. 
+(For more information about Battery assignments, navigate directly to the 
+[Clinical Test Battery](./07_clinical_test_battery.md) section or continue following 
+this order of the).  
+
+### How are Instruments Created?
+
+LORIS instruments can be implemented in one of 2 formats: **LINST** or **PHP**.
+
+**LINST** is a custom LORIS form format which is generated and edited by the LORIS 
+Instrument Builder module. The LINST format allows for the creation of basic forms 
+and can be complemented by a *score* file to allow for automated scoring  and a *rules* 
+file to accommodate for validation of dependencies between form questions and answers.
+
+**PHP** instruments are forms coded manually by a developer in the PHP language. 
+These instruments can also have rules and scoring defined.
+
+Choosing the format in which an instrument is coded is necessary before beginning 
+the process. PHP instruments can eventually be converted to LINST but the opposite is not 
+true. Once an instrument is implemented in LINST format, transforming it to PHP is not 
+trivial. Here are a few bullet points that can help you make your decision.
+
+- ***LINST***
+- *Pros:*
+    - Easy to create (LORIS Instrument Builder)
+    - Ideal for simple instruments
+    - Less prone to bugs and issues
+    - Can be complimented with rules and scoring 
+- *Cons:*
+    - Validation logic can be limited in complex cases
+    - Validation and scoring might still require a developer
+    - Limited flexibility of form elements (default LORIS form elements are used)
+- ***PHP***
+- *Pros:*
+    - Very flexible as form elements can be overwritten, implemented or customized
+    - Allows for more complex validation logic
+    - Can be converted back to LINST (Manual intervention is only required for validation and scoring)
+- *Cons:*
+    - Requires a developer
+    - Significant maintenance required over time to adapt to new LORIS releases
+    - less stable as some elements might change get deprecated over time
+
+> _**Note:** The definitions below can help choosing an instrument format._
+> - _**Validation/Rules:** LORIS allows for 3 levels of form validation in instruments. 
+>    Level 1 consists in setting a form field as required or not to the completion 
+>    of the form. Level 2 set a field to required or not based on the response of 
+>    another field in the form. Level 3 uses custom validation functions to determine 
+>    the validity or the value set for a field. LINST instruments can use levels 1 
+>    and 2, only PHP instruments can use level 3 validation._
+> - _**Scoring:** Scoring in LORIS instruments is always done using the PHP coding 
```

Comment:
Scoring can be done in any language for LINST instruments--it just executes a command in the instrument directory.

Reply:
good catch